userguide/jails.rst

```diff
@@ -1230,17 +1174,15 @@ shows the :guilabel:`Configuration Options` for audiotag.
    Configuration Options for Audiotag
 
 
-This port has five configurable options (DOCS, FLAC, ID3, MP4,
-and VORBIS) and each option is enabled (on) by default.
+This port has five configurable options: *DOCS*, *FLAC*, *ID3*, *MP4*,
+and *VORBIS*. In this port, each option is enabled by defualt.
```

Comment:
Typo.  Also, "In this port" is redundant.  Maybe

"Stars (:literal:`*`) show which options are enabled."